src/metabase/util/malli/schema.clj

```diff
@@ -375,3 +375,8 @@
   [:fn
    {:error/message (format "Collection of %s" item-schema)}
    #(and (coll? %) (every? (partial mc/validate item-schema) %))])
+
+(defn QueryVectorOf
```

Comment:
What does the `Query` prefix mean?

Reply:
It's to parse values from `query` part of request. Weird it's not in the docstring, I'll mention this.